FordLabs/PeopleMover
# Delete archive product message

## Issue
Connects #313 

## What was done
- [x] corrected the logic for the text appearing in the delete product modal

## How to test
1. archived a prodcut and delete it see that only 'Deleting this product will permanently remove it from this space.' appear
2. create a product and delete it. See that 'You can also choose to archive this product to be able to access it later.' also appear in the modal
